src/sentry/tasks/__init__.py

```diff
@@ -0,0 +1,37 @@
+import sentry_sdk
+from django.conf import settings
+
+from sentry.utils import redis
+
+
+class MetricsCluster:
+    def __init__(self, cluster, counter_ttl: int, prefix: str = "symbolicate_event_low_priority"):
+        self.inner = cluster
+        self.counter_ttl = counter_ttl
+        self.prefix = prefix
+
+    def increment_project_event_counter(self, project_id: str, timestamp: float):
+        """Increment the event counter for the given project_id.
+
+        The key is computed from the project_id and the timestamp of the symbolication request, rounded
+        down to the nearest 10 seconds. If the key is not currently set to expire, it will be set to expire
+        in ttl seconds.
```

Comment:
I think the description can be more high level:

The counter is used to track the rate of events for the project, calling this increments the counter of the current time-window bucket with ``timestamp`` providing the time of the event.